fe1-web/src/features/evoting/network/index.ts

```diff
@@ -8,31 +8,51 @@ import {
   handleElectionOpenMessage,
 } from './ElectionHandler';
 import { OpenElection } from './messages/OpenElection';
+import { EvotingConfiguration } from '../objects';
 
 /**
  * Configures the network callbacks in a MessageRegistry.
- *
+ * @param getCurrentLao - A function returning the current load
+ * @param getEventFromId - A function retrieving an event with matching id from the store of the currently active lao
+ * @param addEvent - A function creating a redux action to add a new event to the store of the currently active lao
+ * @param updateEvent - A function returning a redux action for update an event in the currently active lao store
  * @param registry - The MessageRegistry where we want to add the mappings
  */
-export function configureNetwork(registry: MessageRegistry) {
+export const configureNetwork = (
+  getCurrentLao: EvotingConfiguration['getCurrentLao'],
+  getEventFromId: EvotingConfiguration['getEventFromId'],
+  addEvent: EvotingConfiguration['addEvent'],
+  updateEvent: EvotingConfiguration['updateEvent'],
```

Comment:
do we need this here, or can we just pass a `config: EvotingConfiguration` and then use `config.registry`, `config.addEvent`, etc?